ui/desktop.html

```diff
@@ -0,0 +1,532 @@
+        <gmf-map-entrypoint></gmf-map-entrypoint>
+      </div>
+      <gmf-map
+        slot="map"
+        class="gmf-map"
+        gmf-map-map="mainCtrl.map"
+        gmf-map-manage-resize="mainCtrl.manageResize"
+        gmf-map-resize-transition="mainCtrl.resizeTransition"
+      >
+      </gmf-map>
+      <ngeo-query
+        slot="map"
+        active="mainCtrl.queryActive"
+        autoclear="mainCtrl.queryAutoClear"
+        map="::mainCtrl.map"
+      >
+      </ngeo-query>
+      <gmf-floorselector
+        slot="map"
+        class="gmf-floorselector ol-unselectable ol-control"
+        value="mainCtrl.dimensions.FLOOR"
+      >
+      </gmf-floorselector>
+
+      <div
+        slot="infobar-left"
+        ngeo-scaleselector
+        ngeo-scaleselector-map="mainCtrl.map"
+        class="gmf-app-footer-div"
+      ></div>
+      <div slot="infobar-left" id="scaleline" class="gmf-app-footer-div"></div>
+      <gmf-elevationwidget
+        slot="infobar-right"
+        gmf-elevationwidget-map="::mainCtrl.map"
+        gmf-elevationwidget-active="::mainCtrl.options.showInfobar"
+      >
+      </gmf-elevationwidget>
+      <gmf-mouseposition
+        slot="infobar-right"
+        gmf-mouseposition-map="mainCtrl.map"
+        class="text-center gmf-app-footer-div"
+      >
+      </gmf-mouseposition>
+      <ngeo-modal slot="modal" ng-model="mainCtrl.modalShareShown">
+        <gmf-share ng-if="mainCtrl.modalShareShown"></gmf-share>
+      </ngeo-modal>
+      <ngeo-modal slot="modal" ng-model="disclaimerVisibility">
+        <div class="modal-header ui-draggable-handle">
+          <button type="button" class="close" data-dismiss="modal" aria-hidden="true">&times;</button>
+        </div>
+        <div class="modal-body">
+          <div ng-bind-html="disclaimerMsg"></div>
+        </div>
+      </ngeo-modal>
+      <gmf-displayquerygrid
+        slot="footer-queryresult"
+        gmf-displayquerygrid-active="mainCtrl.queryGridActive"
+        gmf-displayquerygrid-map="::mainCtrl.map"
+        ngeo-resizemap="mainCtrl.map"
+        ngeo-resizemap-state="mainCtrl.queryGridActive"
+      >
+      </gmf-displayquerygrid>
+    </gmf-desktop-canvas>
+    <script nomodule>
+      alert(
+        'Your browser is not supported, please use a recent version of Firefox, Chrome or Edge.\n\n' +
+          "Votre navigateur n'est pas supporté, veuillez utiliser une version récente de Firefox, Chrome or Edge.\n\n" +
+          'Ihr Browser wird nicht unterstützt, bitte verwenden Sie eine aktuelle Version von Firefox, Chrome oder Edge.',
+      );
+    </script>
+    <script
+      crossorigin="anonymous"
+      integrity="sha384-RVLCQKq/7QEfq9wE60LncFLNfdYAp/w9yHksCt1tQvZHr0bgMlRhHLd0IrRfSc4T"
+      referrerpolicy="no-referrer"
+      src="https://maps.googleapis.com/maps/api/js?v=3&amp;key=AIzaSyA3NVIy-HOYT0a0CkChA6nFwqEFqHYWBVk"
+    ></script>
+    <script type="module" src="./src/interfaces/ControllerDesktop.ts"></script>
+    <script
```

Comment:
That a sad/weak part... It's okay to merge it, but I wonder if it's not possible to include it in the js build directly.

Reply:
I take many times try to make it to work, it's why I finally I put it here, the issue it that those project don't have sources in es6 modules syntax.